### ISSUE TITLE
Multiomics data integration unveils core transcriptional regulatory networks governing cell-type identity

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#9]
## TL;DR

 Paper presents a computational method, Moni, that systematically integrates epigenetics, transcriptomics, and protein–protein interactions to reconstruct gene regulatory networks (GRNs).

### Paper Link

https://www.nature.com/articles/s41540-020-00148-4

### Author/Institution

Sascha Jung (CIC bioGUNE) & Antonio del Sol (CIC bioGUNE, University of Luxembourg, Basque Foundation for Science, Moscow Institute of Physics and Technology)

## Overview

A plethora of computational approaches have been proposed for reconstructing gene regulatory networks (GRNs) from gene expression data. However, gene regulatory processes are often too complex to predict from the transcriptome alone. Here, we present a computational method, Moni, that systematically integrates epigenetics, transcriptomics, and protein–protein interactions to reconstruct GRNs among core transcription factors and their co-factors governing cell identity.

## Methods

gene regulatory networks

## Cite
Jung, S., del Sol, A. Multiomics data integration unveils core transcriptional regulatory networks governing cell-type identity. npj Syst Biol Appl 6, 26 (2020). https://doi.org/10.1038/s41540-020-00148-4

## Comments

The source code of Moni together with its documentation is available online at https://github.com/saschajung/Moni.
